app/models/data_request.rb

```diff
@@ -56,21 +54,27 @@ class DataRequest < ApplicationRecord
     court: "court",
     cross_borders: "cross_borders",
     cat_a: "cat_a",
-    ndelius: "ndelius",
+    ndelius_contact_lists: "ndelius_contact_lists",
```

Comment:
Changing an old value would at least require a migration of all old values. It's best to be avoided.